src/templates/shared/wrapper.js

```diff
@@ -1,27 +1,28 @@
-const {html, alert, ol, pageAnchor} = require("./bits");
+const {html, alert, escapeHtml, REPO_URL, DISCORD_URL, ul, anchor} = require("./bits");
+const footer = require("./footer");
+const header = require("./header");
+const breadcrumbs = require("./breadcrumbs");
+const toc = require("./toc");
 
-const DISCORD_URL = "https://discord.reclaimers.net";
-const REPO_URL = "https://github.com/Sigmmma/c20";
+const TOC_MIN_HEADERS = 2;
+
+const topLevelTopics = [
+  ["/blam/tags", "Tags"],
+  ["/games/h1", "Halo"]
+];
 
 const STUB_ALERT = alert({type: "danger", body: html`
   <p>🚧 This article is a stub. You can help expand it by submitting content in
   pull requests or issues in this wiki's <a href="${REPO_URL}">source repo</a>.</p>
 `});
 
 const wrapper = (page, metaIndex, body) => {
-  const srcUrl = `${REPO_URL}/tree/master/src/content${page._dirUrl}`;
   const imgAbsoluteUrl = page.img ?
     `${metaIndex.baseUrl}${page._dirUrl}/${page.img}` :
     `${metaIndex.baseUrl}/assets/librarian.png`;
 
-  const breadcrumbs = [];
-  for (let i = 0; i <= page._dir.length; i++) {
-    const parentUrl = "/" + page._dir.slice(0, i).join("/");
-    const parent = metaIndex.pages.find(otherPage => otherPage._dirUrl == parentUrl);
-    if (parent) {
-      breadcrumbs.push(parent);
-    }
-  }
+  const showToc = page.toc !== undefined ? page.toc : page._headers.length > TOC_MIN_HEADERS;
```

Comment:
If you really want to force the ToC to appear or not on the page, you can provide in the page's YAML front-matter: `toc: true` or `toc: false`.